### PR TITLE
Prepare links for migration

### LINKED ISSUE #22: Prepare links in the bottom part
The idea here is to prevent conflicts. Even if two teams work on a different page/file, they will have to update the links so that their page can be accessed. However, by doing this, they will change the same file which may lead to a Git conflict (quite a bad experience for a first experimentation with Git....even for advanced users anyway).

### PR DESCRIPTION
**Pull request by gmantele**
This PR updates links of the sitemap (list of links available at the bottom of all website pages). All links (except Wiki and Documents links) are now made relative to this prototype website.

Don't hesitate to tell me if I forgot some or if I fixed the wrong one or in the wrong way.

FIxes #22

### CODE REVIEW

**Comment by gmantele** [comment]
I may still change one breaking thing:

- move the content of the `content/pages/` directory to the upper level (i.e. `content/`)
- remove the `pages/` directory

Otherwise, URLs would be `https://ivoa.net/pages/about/`....which would be weird and would break any bookmark on existing pages that people might have kept on their machine. So, with this last modification, we would have the URL: `https://ivoa.net/about/`.

Just tell me if it is a bad idea.

**Comment by gmantele** [comment]
Ok.According to me, this PR is now ready for review and to be merged.

File: layouts/partials/sitemap.html [hunk 1]
```diff
@@ -26,12 +26,12 @@
...
-                    <li><a href="https://www.ivoa.net/astronomers/vo_glossary.html">VO Glossary</a></li>
-                    <li><a href="https://www.ivoa.net/astronomers/applications.html">VO Applications</a></li>
-                    <li><a href="https://www.ivoa.net/newsletter/index.html">IVOA newsletter</a></li>
-                    <li><a href="https://www.ivoa.net/astronomers/vo_for_public.html">VO for Students & Public</a></li>
+                    <li><a href="{{ .Site.Home.Permalink}}/astronomers/getting_started/">Getting Started</a></li>
+                    <li><a href="{{ .Site.Home.Permalink}}/astronomers/using_the_vo/">Using the VO</a></li>
+                    <li><a href="{{ .Site.Home.Permalink}}/astronomers/vo_glossary/">VO Glossary</a></li>
+                    <li><a href="{{ .Site.Home.Permalink}}/astronomers/applications/">VO Applications</a></li>
+                    <li><a href="{{ .Site.Home.Permalink}}/newsletter/">IVOA newsletter</a></li>
+                    <li><a href="{{ .Site.Home.Permalink}}/astronomers/vo_for_public/">VO for Students & Public</a></li>
```
Comment (molinaro-m) [inline layouts/partials/sitemap.html:34]: I think these all have an extra **"/"** between the `{{ .Site.Home.Permlink}}`  and the folder. It works in the browser anyway, but maybe we can remove it.

**Review by molinaro-m — APPROVED** [review]
Besides the small comment, I thin this is fine.
I checked the version's outcome on the webstage and looks fine.

**Review by JeremyMcCormick — APPROVED** [review]
Looks good to me. The link for the one ported page works fine and the others are 404 now but I assume that is intended here so we can work on adding the new pages that are pointed to now.

We need to update a few things in the contributing guide based on these changes, but I can do that on another branch.

File: layouts/partials/sitemap.html (hunk 1 above)
Comment (gmantele) [reply layouts/partials/sitemap.html:34]: Yes, I have noticed that too. As it does not seem to be an issue, I have not put an high priority to solve this yet. But I can have a quick look now, if you want to.

Comment (molinaro-m) [reply layouts/partials/sitemap.html:34]: no, definitely no need to prioritise this, let's just keep it in mind

**Pull request merged**